verification/cbmc/proofs/aws_hash_table_remove/Makefile

```diff
@@ -24,14 +23,14 @@ PROOF_SOURCES += $(PROOF_STUB)/error.c
 PROOF_SOURCES += $(PROOF_STUB)/memset_override_no_op.c
 PROOF_SOURCES += $(PROOF_STUB)/s_emplace_item_override.c
 PROOF_SOURCES += $(PROOF_STUB)/s_expand_table_override.c
+PROOF_SOURCES += $(PROOF_STUB)/s_remove_entry_override.c
```

Comment:
`ABSTRACTIONS`

Reply:
I have an issue to correct this (https://github.com/awslabs/aws-c-common/issues/712);